docs/guide/connecting.asciidoc

```diff
@@ -336,6 +338,20 @@ es = Elasticsearch(
 )
 ----
 
+Connecting to a cloud instance with an API Key: 
+https://www.elastic.co/guide/en/observability/current/beats-api-keys.html#beats-api-keys
+
+[source,python]
+----
+# Create the client instance
+client = Elasticsearch(
+    # For local development
+    # hosts=["http://localhost:9200"] 
+    cloud_id=CLOUD_ID,
+    api_key=API_KEY,
+)
+----
+
```

Comment:
I'm not sure I'm following, how is this different from above? That said, an example with basic auth would be useful, since this is what the Elastic Cloud onboarding suggest. Maybe we should add this here?